main.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"log"
+
+	"context"
+
+	"github.com/shomali11/slacker"
+)
+
+func main() {
+	var config Config
+	err := config.FromEnv()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	bot := slacker.NewClient(config.Slack.Token)
+	// defining which function handles the bot Init phase
+	bot.Init(
```

Comment:
bot commands should be initialized in the separate function, not main.

Reply:
Done.